src/components/timeline/Markers.js

```diff
@@ -162,14 +333,19 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
     const mouseDownItem = this._hitTest(e);
     this.setState({ mouseDownItem });
     if (mouseDownItem !== null) {
-      if (e.target.setCapture) {
-        e.target.setCapture();
+      // Disabling Flow type checking because Flow doesn't know about setCapture.
+      const canvas = (e.currentTarget: any);
+      if (canvas.setCapture) {
+        // This retargets all mouse events to this element. This is useful
+        // when for example the user releases the mouse button outside of the
+        // browser window.
+        canvas.setCapture();
```

Comment:
This change was made necessary, I think that's because `_hitTest` changed, which in turn makes `e` better typed.